srcpkgs/wpan-tools/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'wpan-tools'
+pkgname=wpan-tools
+version=0.9
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config libnl3-devel"
+makedepends="libnl3-devel"
+short_desc="Userspace tools for Linux IEEE 802.15.4 stack"
+maintainer="Paige Thompson <paige@paige.bio>"
+license="ISC"
+homepage="https://linux-wpan.org"
+distfiles="https://github.com/linux-wpan/wpan-tools/releases/download/wpan-tools-${version}/wpan-tools-${version}.tar.gz"
+checksum=fa76d9c1874220e4b1f91c226f42baf1e372ea8ccf4b892effaf0d164448f608
+
+post_install() {
+	vlicense COPYING
```

Comment:
That's only a symlink in their repo, it'd probably better to directly vlicense https://github.com/linux-wpan/wpan-tools/blob/master/LICENSES/ISC.txt I think

Reply:
alright

Reply:
lol dude 
```
/Users/paigethompson/wpan-tools-0.9
❯ tree
.
├── COPYING
├── Makefile.am
├── Makefile.in
├── aclocal.m4
├── build-aux
│   ├── compile
│   ├── config.guess
│   ├── config.sub
│   ├── depcomp
│   ├── install-sh
│   ├── ltmain.sh
│   └── missing
├── config.h.in
├── configure
├── configure.ac
├── examples
│   ├── Makefile.am
│   ├── Makefile.in
│   ├── README.examples
│   ├── TODO
│   ├── af_ieee802154_rx.c
│   ├── af_ieee802154_tx.c
│   ├── af_inet6_rx.c
│   ├── af_inet6_tx.c
│   ├── af_packet_rx.c
│   └── af_packet_tx.c
├── m4
│   ├── libtool.m4
│   ├── ltoptions.m4
│   ├── ltsugar.m4
│   ├── ltversion.m4
│   └── lt~obsolete.m4
├── src
│   ├── Makefile.am
│   ├── Makefile.in
│   ├── info.c
│   ├── interface.c
│   ├── iwpan.c
│   ├── iwpan.h
│   ├── mac.c
│   ├── nl802154.h
│   ├── nl_extras.h
│   ├── phy.c
│   └── sections.c
├── wpan-hwsim
│   ├── Makefile.am
│   ├── Makefile.in
│   ├── mac802154_hwsim.h
│   └── wpan-hwsim.c
└── wpan-ping
    ├── Makefile.am
    ├── Makefile.in
    ├── README.wpan-ping
    └── wpan-ping.c
```